ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,1180 @@
+     */
+    public renderSingleButton(this: Breadcrumbs): void {
+        const breadcrumbs = this,
+            chart = breadcrumbs.chart,
+            breadcrumbsList = breadcrumbs.breadcrumbsList,
+            breadcrumbsOptions = breadcrumbs.options,
+            lastBreadcrumbs = breadcrumbsList[breadcrumbsList.length - 2] &&
+                breadcrumbsList[breadcrumbsList.length - 2][2],
+            buttonSpacing = breadcrumbsOptions.buttonSpacing;
+
+        // Draw breadcrumbs.
+        if (breadcrumbsList) {
+            // Inital position for calculating the breadcrumbsGroup.
+            let posX: number = lastBreadcrumbs ? lastBreadcrumbs.x +
+                    lastBreadcrumbs.element.getBBox().width + buttonSpacing : 0;
+            const posY: number = buttonSpacing;
+
+
+            const previousBreadcrumb = breadcrumbsList[breadcrumbsList.length - 2],
+                drilldownLevels = chart.drilldownLevels;
+
+            if (!chart.drillUpButton &&
+                (
+                    (drilldownLevels && drilldownLevels.length) ||
+                    this.level > 0
+                )
+            ) {
+                chart.drillUpButton = breadcrumbs.renderButton(previousBreadcrumb, posX, posY);
+            } else if (chart.drillUpButton) {
+                // Update button.
+                this.updateSingleButton();
+            }
+        }
+    }
+
+    /**
+     * Update group position based on align and it's width.
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#renderSingleButton
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public alignBreadcrumbsGroup(this: Breadcrumbs): void {
+        const breadcrumbs = this,
+            breadcrumbsOptions = breadcrumbs.options,
+            positionOptions = breadcrumbsOptions.position,
+            alignTo = (
+                breadcrumbsOptions.relativeTo === 'chart' ||
+                breadcrumbsOptions.relativeTo === 'spacingBox' ?
+                    null :
+                    'scrollablePlotBox'
+            ),
+            bBox = breadcrumbs.breadcrumbsGroup.getBBox(),
+            width = bBox.width,
+            height = bBox.height;
+
+
+        let translationX = 0 - bBox.x;
+        let translationY = 0;
+
+        // Default setting
+        if (breadcrumbsOptions.relativeTo === 'plotBox') {
+            if (positionOptions.verticalAlign === 'bottom') {
+                translationY += height + bBox.y;
+            } else if (positionOptions.verticalAlign !== 'middle') {
+                translationY -= height + bBox.y;
+            }
+        } else if (positionOptions.verticalAlign === 'bottom') {
+            translationY -= height + bBox.y;
+        }
+
+        if (positionOptions.align === 'right') {
+            translationX = -width;
+        } else if (positionOptions.align === 'center') {
+            translationX = -width / 2;
+        }
```

Comment:
Q: Do we need all this logic related to `translationX` and `translationY`? Other aligned groups (Legend, ResetZoomButton, etc.) seems to be fine without extra logic, just using `align()`.

Reply:
Legend is checking doing something similar:
  `alignTo.y += rangeSelector.getHeight();`

The other difference is that legend is relative to the chart and breadcrumbs are not - because of that maybe the functionality is slightly different. We may investigate it further though.

Reply:
Hint: Use `width` and `height`